src/21kHz.cpp

```diff
@@ -1,19 +1,12 @@
 #include "21kHz.hpp"
 
-
-Plugin *plugin;
-
+Plugin *pluginInstance;
 
 void init(Plugin *p) {
```

Comment:
Indentation looks a bit weird in this method!

Reply:
I agree. This is not the only file with indentation issues. Some lines were hard tabbed and some soft tabbed. This is not something I did to the code but rather was there before. If you view the code in a proper editor (I use Atom), physical screen spacing is compensated and all looks fine. If I end up adopting this module, I'll clean up all the soft/hard indentation mess.

Reply:
@netboy3, ate you still considering adopting this module? I would love to see some builds as I haven't made the time to setup a VCV Rack development environment. 

I think I am going to send 21kHz an email. I got his email off of the VCV Rack website. It's jakehuryn@gmail.com if you care to send him one too. I hope it's not dead like his Github account. :sweat_smile: